src/react_native/hooks.cljs

```diff
@@ -10,3 +12,21 @@
 (defn use-back-handler
   [handler]
   (.useBackHandler hooks handler))
+
+(defn use-interval
+  [cb cleanup-cb delay]
+  (let [saved-callback (rn/use-ref)]
+    (rn/use-effect
+     (fn []
+       (oops/oset! saved-callback "current" cb))
+     [cb])
+
+    (rn/use-effect
+     (fn []
+       (let [tick #((oops/oget saved-callback "current"))]
```

Comment:
`tick` can be simplified:

`#((oops/oget saved-callback "current"))` => `(oops/oget saved-callback "current")`

I think both will behave the same